src/modules/numbers/dto/get-numbers-query.dto.ts

```diff
@@ -5,12 +5,12 @@ export class GetNumbersQueryDto {
   @IsInt()
   @IsNotEmpty()
   @Max(9)
```

Comment:
Can we also add `@Min(2)`, or a number you think is a common min standard please?

Reply:
Same as above, Added @min(1).